sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -34,7 +34,7 @@ This table shows the relationship between SDK versions and supported API version
 
```

Comment:
if we updating the GA stuff now, should we remove the `--pre` as well?

Reply:
Yeah I'll do that now, I'll probably still have another few docs updates PRs just a heads up. Also I realized that the release date it actually not going to be the 6th, asking the crew and will update. I think it might be the 8th.

Reply:
JFYI, there's a `--pre` on the samples README too

Reply:
Thank you! Done